Web.Framework/HttpModel.cs

```diff
@@ -80,13 +82,20 @@ private static string CombineRoute(string prefix, string template)
         }
     }
 
-    public class MethodModel
+    public class MethodModel : IEndpointConventionBuilder
```

Comment:
So this an example where you'd want a `T`. Because you want to be able to perform operations on this boy.

Reply:
Yeah, this was a point I made during **Endpoint Con 2108 Summer Gathering**. It's basiclaly java with all of the `<? super T>`

Reply:
It's a gotcha for anyone writing extension methods.